README.md

```diff
@@ -2,15 +2,20 @@
 
 NOT ready for use.
 
-When it is, usage will look like:
+Current test procedure:
 
-```
-cd lib/modules/whatever
-apostrophe-3-upgrade-module index.js
-```
+* `git clone` this module
+* `npm install`
+* `cd` to your project
+* make sure `git status` is clean
+* make a **new branch** for the migration
+* make sure you are cd'd into your project
+* `~/apostrophecms/apostrophe-3-upgrade-tools .`
```

Comment:
I had to use `node ~/mypath/apostrophe-3-upgrade-tools/app.js .` and I also changed the `main` property of the package.json file to be `app.js`. Before I was getting a permission error.

Reply:
I took this opportunity to fix all that, rename the repo and take a stab at what the CLI command should really be called. You can test drive it after you do `npm link`.